app/models/sample.rb

```diff
@@ -282,6 +282,13 @@ def release
     positive: 1
   }
 
+  enum priority: {
+    nopriority: 0,
+    backlog: 1,
+    surveillance: 2,
+    priority: 3
+  }
+
```

Comment:
Looks good. Only small comment would be that no_priority is a bit easier to read than nopriority, so could go for that for the enum value instead?